MMA Removal Utility/README.md

```diff
@@ -262,6 +336,56 @@ let virtualMachinesWithMMAExtensions = virtualMachines
 | project VMResourceID, OSType, ExtensionResourceId, ExtensionType;
 virtualMachinesWithMMAExtensions
 ```
+
+For VMSS(s) having MMA agent present:
+
+``` KQL
+let timeago = timespan(7d);
+let vmss = Inventory_CL
+| where TimeGenerated > ago(timeago)
+| where ResourceType contains "VirtualMachineScaleSets"
```

Comment:
As suggested earlier avoid contains check, if =~ operator can be utilized.

Reply:
Added =~ in all the places and tested the query in LA